zoia_lib/UI/ZOIALibrarian_ps.py

```diff
@@ -259,7 +259,7 @@ def create_dwn_btn(self, i, idx):
         dwn.setFont(self.ui.table_PS.horizontalHeader().font())
         dwn.clicked.connect(self.initiate_download)
         # Only enable it if we haven't already downloaded the patch.
-        if idx in os.listdir(self.path):
+        if idx in os.listdir(self.path) or idx == 169172 or idx == '169172':
             dwn.setEnabled(False)
             dwn.setText("Downloaded")
```

Comment:
This works but isn't exactly ideal (hardcoding patch IDs for exclusion could potentially lead to unexpected problems if PS ever changes these numbers on their end). 

From what I gather from Discord this is because contains no actual 'patch' but rather a 3D printer file right? I support your suggestion of contacting PS. When I was working on the initial version I excluded questions for this same reason (because there was nothing to download), so maybe an additional classification could be added that we could then avoid on our end to prevent hardcoded IDs.